src/mrack/providers/podman.py

```diff
@@ -37,6 +37,7 @@ def __init__(self):
         """Initialize provider."""
         self._name = PROVISIONER_KEY
         self.dsp_name = "Podman"
+        self.max_attempts = 1 # for retry strategy
```

Comment:
`max_attempts` or `max_retry`? I got confused what providers other than beaker are expecting on prov-config file.

Reply:
yeah we need to change it to max_retry